src/System.Private.CoreLib/shared/System/AggregateException.cs

```diff
@@ -440,26 +440,14 @@ public override string Message
             }
         }
 
-        /// <summary>
-        /// Creates and returns a string representation of the current <see cref="AggregateException"/>.
-        /// </summary>
-        /// <returns>A string representation of the current exception.</returns>
-        public override string ToString()
+        protected internal override string InnerExceptionToString(Exception? inner = null)
         {
-            StringBuilder text = new StringBuilder();
-            text.Append(base.ToString());
-
-            for (int i = 0; i < m_innerExceptions.Count; i++)
+            StringBuilder sb = StringBuilderCache.Acquire();
```

Comment:
I do not think that switching to StringBuilderCache is an improvement here. This is not performance sensitive path and the string is very likely be over the StringBuilderCache limit.

Reply:
Remove from Message above also?

Reply:
I'll leave it. That will sometimes be in the limit.